VCFv4.3.tex

```diff
@@ -311,8 +311,7 @@ \subsubsection{Fixed fields}
 
   \item ALT --- alternate base(s): Comma separated list of alternate non-reference alleles.
   These alleles do not have to be called in any of the samples.
-  Options are base Strings made up of the bases A,C,G,T,N,*, (case insensitive) or a MISSING value `.' (no variant) or an angle-bracketed ID String (``$<$ID$>$'') or a breakend replacement string as described in the section on breakends.
-  The `*' allele is reserved to indicate that the allele is missing due to an overlapping deletion.
+  Options are base Strings made up of the bases A,C,G,T,N (case insensitive) or the `*' symbol (allele missing due to overlapping deletion) or a MISSING value `.' (no variant) or an angle-bracketed ID String (``$<$ID$>$'') or a breakend replacement string as described in the section on breakends.
```

Comment:
> allele missing due to overlapping deletion

This should be changed to "allele missing due to other overlapping alleles", not limited to deletions. For example
```txt
chr1  100  A  G
chr1  100  A  T,*
```
should be technically allowed.

Reply:
> This should be changed to "allele missing due to other overlapping alleles", not limited to deletions

I agree with this since it would allow the option of using `0` as an assertion of reference. Possibly replace "missing" with "unspecified" to avoid confusion with `.`?